apps/frontend/src/components/global/groups/GroupModal.vue

```diff
@@ -45,6 +45,13 @@
               </v-tooltip>
             </v-col>
           </v-row>
+          <v-textarea
+            v-model="groupInfo.desc"
+            data-cy="desc"
+            label="Description"
+            rows="1"
+            auto-grow
```

Comment:
what happens when you have a giant description for some reason?

Reply:
It'll just expand properly, hence the "auto-grow"